content/en/docs/concepts/workloads/pods/init-containers.md

```diff
@@ -282,9 +282,7 @@ define readiness distinct from completion. This is enforced during validation.
 
 Use `activeDeadlineSeconds` on the Pod to prevent init containers from failing forever.
 The active deadline includes init containers.
-However it is recommended to use `activeDeadlineSeconds` if user deploy their application
```

Comment:
this seems the correct explanation for `activeDeadlineSeconds`. no change needed here.